llvm/lib/Target/X86/X86.h

```diff
@@ -201,6 +202,7 @@ void initializeX86ReturnThunksPass(PassRegistry &);
 void initializeX86SpeculativeExecutionSideEffectSuppressionPass(PassRegistry &);
 void initializeX86SpeculativeLoadHardeningPassPass(PassRegistry &);
 void initializeX86TileConfigPass(PassRegistry &);
+void initializeX86MulAddPassPass(PassRegistry &);
```

Comment:
Please rewrite plugin as dyn lib. You may look into #99 as example.

Reply:
> Please rewrite plugin as dyn lib. You may look into #99 as example.

done ✅